pandas/tests/groupby/aggregate/test_aggregate.py

```diff
@@ -1413,3 +1413,14 @@ def test_multi_axis_1_raises(func):
     gb = df.groupby("a", axis=1)
     with pytest.raises(NotImplementedError, match="axis other than 0 is not supported"):
         gb.agg(func)
+
+
+def test_agg_of_list():
+    # GH#25581
+    df1 = DataFrame([[20, "A"], [20, "B"], [30, "C"]])
```

Comment:
Could you add the other cases from the issue too?

Reply:
From the issue there are 3 other test cases than the one I originally added.  I can add 2 of them, but the final test case still fails on the master with a different error message than in 1.4.1.